### ISSUE TITLE
Additional funding to apply for

### ISSUE THREAD

**Issue by lwjohnst86** [UofTCoders/council#31]
(just so I don't forget hihi)

Student initiative fund: https://www.studentlife.utoronto.ca/bts/sif

Might be useful to apply for if we ever want to do a University-wide 'Hackathon' or something.

**Comment by linamnt** [comment]
https://www.utgsu.ca/funding/grants/ 
The special projects one is what I applied to last year, good for some snack money!

**Comment by joelostblom** [comment]
From #124 : In the previous student club I was part of, we applied for funding from many places within UofT. It was quite a lot of work, but we also got a substantial amount of money. Is this something we are interested in and are we eligible for anything although we primarily have grad students and not undergrads?

Something like this for the engineering faculty http://mse.utoronto.ca/current/student-life/funding/
